js/controllers/MenuHandler.js

```diff
@@ -79,18 +82,41 @@
 
 
 	/**
-	 * Attach a class to any dropdown menus that will stray off-screen to align
-	 * them to the right edge of their parent
+	 * Check if submenus are straying off-screen and adjust as needed
 	 */
 	$.pkp.controllers.MenuHandler.prototype.setDropdownAlignment = function() {
-		var width = Math.max(
-				document.documentElement.clientWidth, window.innerWidth || 0);
+		var $this = $(this),
+			width = Math.max(
+				document.documentElement.clientWidth, window.innerWidth || 0),
```

Comment:
If I were doing this, I'd run the linting tool on my own system -- but it's a bit of a PITA to set up. Details on the wiki, and in `.travis.yml`, and you'll need to stick with the same versions of the tools that Travis uses if you want to get the same results. So a submodule commit is an easier solution -- no need to send a PR if you've got Travis hooks enabled in your own github account.